### PR TITLE
Add new paper issue templates as appendices

### LINKED ISSUE #712: Add paper review templates as appendices
**Is your feature request related to a problem? Please describe.**
In the methods, we mention the issue templates for reviewing papers. These need to be added as appendices.

**Describe the solution you'd like**
Need to copy the text into new content for the manuscript.

**Describe alternatives you've considered**

**Additional context**
Thanks to @agitter for catching this in #698

### PR DESCRIPTION
**Pull request by agitter**
### Description of the proposed additions or changes

Adds appendices B-D

### Related issues

Closes #712

### Suggested reviewers (optional)

@rando2 

### Checklist

- [x] Text is formatted so that each sentence is on its own line.

- [x] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

Let's see how Manubot renders this.  I may need to remove the Markdown headers to make it look better.

### CODE REVIEW

**Comment by agitter** [comment]
The AppVeyor preview build above looked very bad.  I'm trying again, this time removing the subsection headers.

File: build/templates-appendix.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# Uses sed command to strip Markdown frontmatter from
+# https://stackoverflow.com/questions/28221779/how-to-remove-yaml-frontmatter-from-markdown-files
+
+appendix=content/96-template-appendix.md
+
+echo "Adding diagnostics template to appendix"
+echo -e "## Appendix B {.page_break_before}\n" > $appendix
```
Comment (rando2) [inline build/templates-appendix.sh:8]:
```suggestion
echo -e "## Appendix B {.page_break_before}\nContributors were asked to complete this template to summarize and evaluate new papers related to diagnostics.\n" > $appendix
```

File: build/templates-appendix.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+sed '1{/^---$/!q;};1,/^---$/d' .github/ISSUE_TEMPLATE/new-diag-study-template.md | sed 's/^#*\ //' >> $appendix
+
+echo "Adding therapeutics template to appendix"
+echo -e "\n## Appendix C {.page_break_before}\n" >> $appendix
```
Comment (rando2) [inline build/templates-appendix.sh:12]:
```suggestion
echo -e "\n## Appendix C {.page_break_before}\nContributors were asked to complete this template to summarize and evaluate new papers related to therapeutics.\n" >> $appendix
```

File: build/templates-appendix.sh [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+sed '1{/^---$/!q;};1,/^---$/d' .github/ISSUE_TEMPLATE/new-therapy-study-template.md | sed 's/^#*\ //' >> $appendix
+
+echo "Adding general paper template to appendix"
+echo -e "\n## Appendix D {.page_break_before}\n" >> $appendix
```
Comment (rando2) [inline build/templates-appendix.sh:16]:
```suggestion
echo -e "\n## Appendix D {.page_break_before}\nContributors were asked to complete this template to summarize and evaluate new papers related to topics besides therapeutics and diagnostics.\n" >> $appendix
```

**Review by rando2 — APPROVED** [review]
Just an idea to add a sentence at the top defining what each template was used for!

**Comment by agitter** [comment]
> Just an idea to add a sentence at the top defining what each template was used for!

Good idea, I accepted and reran the script.

I'm happy with how the appendices look now.  There isn't much formatting, but I think simpler is better so we don't have all of the headings clutter the table of contents.

**Review by rando2 — APPROVED** [review]

**Pull request merged**